benchmark/scripts/Benchmark_Driver

```diff
@@ -227,24 +236,28 @@ def submit(args):
     submit_to_lnt(data, args.lnt_host)
     return 0
 
+
 def run(args):
     optset = args.optimization
     file = os.path.join(args.tests, "Benchmark_" + optset)
     run_benchmarks(file, benchmarks=args.benchmarks,
-            num_samples=args.iterations, verbose=True,
-            log_directory=args.output_dir,
-            swift_repo=args.swift_repo)
+                   num_samples=args.iterations, verbose=True,
+                   log_directory=args.output_dir,
+                   swift_repo=args.swift_repo)
```

Comment:
This alignment style is not very compatible with 80 columns or 4-space indentation.  I would prefer we don't adopt it.